model_compression_toolkit/metadata.py

```diff
@@ -43,13 +43,16 @@ def create_model_metadata(tpc: TargetPlatformCapabilities,
 def get_versions_dict(tpc) -> Dict:
     """
 
-    Returns: A dictionary with TPC and MCT versions.
+    Returns: A dictionary with TPC, MCT and TPC-Schema versions.
 
     """
     # imported inside to avoid circular import error
     from model_compression_toolkit import __version__ as mct_version
-    tpc_version = f'{tpc.name}.{tpc.version}'
-    return {MCT_VERSION: mct_version, TPC_VERSION: tpc_version}
+    tpc_version = f'{tpc.tp_model.tpc_minor_version}.{tpc.tp_model.tpc_patch_version}'
```

Comment:
I know we said to concat the minor and patch, but on second thought, it would be better to just save to separate fields